samcli/local/docker/container.py

```diff
@@ -17,13 +20,21 @@
 
 LOG = logging.getLogger(__name__)
 
+START_CONTAINER_TIMEOUT = float(os.environ.get("SAM_CLI_START_CONTAINER_TIMEOUT", 5))
+
 
 class ContainerResponseException(Exception):
     """
     Exception raised when unable to communicate with RAPID APIs on a running container.
     """
 
 
+class ContainerStartTimeoutException(Exception):
+    """
+    Exception raised when timeout was reached while starting a container.
+    """
+
+
```

Comment:
new exception because we don't want to reuse another exception and accidentally throw an exit code of zero when we don't have to (we're only doing it on the invoke to match the previous behaviour)